front/package.json

```diff
@@ -167,7 +167,7 @@
     "start": "import-meta-env-prepare -x .env.example && vite",
     "build": "vite build",
     "test": "import-meta-env-prepare -x .env.example && vitest",
-    "lint": "eslint --ext .ts,.tsx,.js,.jsx src --max-warnings 69",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx src --max-warnings 40",
```

Comment:
```suggestion
    "lint": "eslint --ext .ts,.tsx,.js,.jsx src --max-warnings 18",
```

Reply:
ok and added "@typescript-eslint/no-explicit-any": 2,